### DISCUSSION TITLE
Einzelgetränkausgabe im Maker Tab

### DISCUSSION THREAD

**Discussion by Marvin0795** [AndreWohnsland/CocktailBerry#146] [Ideas]
Säfte als Einzelgetränkausgabe im Maker Tab.

**Comment by AndreWohnsland** [comment]
Hi, ein ziemlich solches Feature gibt es schon in dem Zutaten Tab/Reiter. Meinst du so etwas, oder stellst du dir was anderes vor?

**Reply by Marvin0795** [reply]
Hallo Andre und danke für deine schnelle Rückmeldung! 
Wir haben die Seiten gesperrt gehabt und viele wollten ganz gern einen einzelnen Saft haben. Man kann natürlich auch ein Rezeot erstellen. War nur so eine Idee.
![IMG_4332](https://github.com/AndreWohnsland/CocktailBerry/assets/102437306/aa512a04-c5d0-4f62-a46c-293ef096154b)

**Reply by AndreWohnsland** [reply]
Ja, hab auch schon dran gedacht das ggf. auf den Flaschen tab zu machen. Oder einen    ‚Cocktail‘ Standard mäßig der „einzelne Zutat“ oder so heißt und dann das Fenster auf macht. Ich grübel mal drüber nach.

**Reply by Marvin0795** [reply]
Ja das hört sich gut an. Sind ja alles nur Ideen und Anregungen, die bei der ersten Party mit der Maschine in den Sinn gekommen sind 👍

**Reply by AndreWohnsland** [reply]
Immer gerne, davon leben Projekte, dass auch aktive Anmerkungen kommen. Die Machbarkeit & Umsetzbarkeit ist ja dann in meiner Hand ^^
